zebra/kernel_socket.c

```diff
@@ -245,9 +247,10 @@ static inline int af_check(int family)
 	return 0;
 }
 
-size_t rta_get(caddr_t sap, void *destp, size_t destlen)
+size_t _rta_get(caddr_t sap, void *destp, size_t destlen, bool checkaf)
```

Comment:
Could this be 'static', since it's an internal/utility function? Also, I don't love the little leading underscore name - could the name be a bit clearer, like 'rta_get_helper' or 'rta_get_internal' ?